raiden-ts/src/raiden.ts

```diff
@@ -592,15 +592,31 @@ export class Raiden {
     assert(Address.is(partner), [ErrorCodes.DTA_INVALID_ADDRESS, { partner }], this.log.info);
     const tokenNetwork = await this.monitorToken(token);
     assert(!options.subkey || this.deps.main, ErrorCodes.RDN_SUBKEY_NOT_SET, this.log.info);
-    const deposit = options.deposit === undefined ? undefined : decode(UInt(32), options.deposit);
+
+    // Note that we use the advantage of the UInt decoding here, but immediately
+    // convert it to a plain number again.
+    const settleTimeout =
+      options.settleTimeout === undefined
```

Comment:
```suggestion
      !options.settleTimeout
```
Only other falsy values are also invalid (`0`, `''`)

Reply:
I was actually re-using your code. :laughing: 
But sure it is an improvement. I guess you could argue that the user should be forced to be more explicit, but...

Reply:
:heavy_check_mark: